### PR TITLE
README: update reviewers and project status

### PR DESCRIPTION
**Pull request by myungjoo**
- Added the two new reviewers
- Added more info on the project status

Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,6 +90,17 @@ For more details, please access the following manuals.
...
+## AI Acceleration Hardware Support
+
+Although a framework may accelerate transparently as Tensorflow-GPU does, nnstreamer provides various hardware acceleration subplugins.
+- Movidius-X via ncsdk2 subplugin: Released
+- Movidius-X via openVINO subplugin: Released
+- Edge-TPU via edgatpu subplugin: Released
+- NNRuntime via nnfw subplugin: Released
+- Verisilicon-Vivante via vivante subplugin: WIP
+- Qualcomm SNPE: WIP
+- Exynos: WIP
```
Comment (kparichay) [inline README.md]: ArmNN: WIP?

File: README.md [hunk 2]
```diff
@@ -86,6 +90,17 @@ For more details, please access the following manuals.
 - [CI service status](http://nnsuite.mooo.com/)
 - [TAOS-CI config files for nnstreamer](.TAOS-CI).
 
+## AI Acceleration Hardware Support
+
+Although a framework may accelerate transparently as Tensorflow-GPU does, nnstreamer provides various hardware acceleration subplugins.
+- Movidius-X via ncsdk2 subplugin: Released
+- Movidius-X via openVINO subplugin: Released
+- Edge-TPU via edgatpu subplugin: Released
```
Comment (kparichay) [inline README.md]: edgaTPU->edgeTPU

File: README.md (hunk 1 above)
Comment (myungjoo) [reply README.md]: Added

File: README.md (hunk 2 above)
Comment (myungjoo) [reply README.md]: Fixed

**Pull request merged**